### PR TITLE
Merging data generation script into my master branch

### PR DESCRIPTION
**Pull request by Ashcoca**
@CZosa @TonyWu5 @brenden720 

Code review if you get a chance! If you're busy don't sweat it, there isn't much to look over. Mostly just the server endpoints and the data generation script. I know I'm missing the server endpoint for the add a review, I just noticed I hadn't finished that, I'll get that wrapped up soon!

Working on getting my databases seeded now. Did you know Cassandra doesn't work on Windows? Fun fact.

### CODE REVIEW

File: database/NEWseed.js [hunk 1]
```diff
@@ -3,15 +3,11 @@ const path = require('path');
...
-
-const filePath = path.join(__dirname, 'data.tsv');
-
-
-const userCount = 10000000;
+const filePath = path.join(__dirname, 'data.csv');
 
 const userData = () => {
-  fs.writeFileSync(filePath, '\n');
+  const userCount = 1000
```
Comment (CZosa) [inline database/NEWseed.js:9]: add a semi-colon here

**Review by CZosa — COMMENTED** [review]
Hey Ash, your code looks polished. Only thing I found was a missing semi-colon on line 9. I noticed that you used 1000 for the userCount. Maybe check to see if you can use 10M records?

File: server/index.js [hunk 2]
```diff
@@ -63,13 +63,15 @@ app.put('/:courseId/reviews/:reviewId', (req, res) => {
 
 // app.delete to delete a review
 // do I need to decrement the matching user review count?
+// consider migrating this functionality to server models for unity 
 app.delete('/:courseId/reviews/:reviewId', (req, res) => {
   let review = req.body
```
Comment (CZosa) [inline server/index.js:68]: add semi-colon

**Review by CZosa — COMMENTED** [review]
please see comment

File: server/index.js (hunk 2 above)
Comment (Ashcoca) [reply server/index.js:68]: Thanks Chris!

**Pull request merged**